src/target/riscv/riscv.c

```diff
@@ -611,13 +622,43 @@ static int jim_report_ebreak_config(const struct riscv_private_config *config,
 	return JIM_OK;
 }
 
+static int jim_configure_cetrig(struct riscv_private_config *config,
+		struct jim_getopt_info *goi)
+{
+	if (goi->argc == 0) {
+		Jim_WrongNumArgs(goi->interp, 1, goi->argv - 1,
+				"?disable|enable?");
+		return JIM_ERR;
+	}
+
+	/* Here a dcsr field is processed, e.g: riscv.cpu configure -mprven enable" */
```

Comment:
This comment mentions `-mprven enable` but the function is dealing with `-cetrig`.

Perhaps it is a copy-paste mistake?